app/templates/components/paper-autocomplete.hbs

```diff
@@ -40,6 +40,7 @@
       onKeydown=(action "onKeydown")
       searchEnabled=(readonly searchEnabled)
       searchField=(readonly searchField)
+      searchText=searchText
```

Comment:
Can we be explicit here and show that's searchText is one way with readonly helper